### PR TITLE
LENS-44 - [UI] Adapt notification page styles

### PR DESCRIPTION
**Pull request by mlallai**
## What does this PR do?

[UI] Adapt notification page styles

## Related ticket

Fixes [LENS-44](https://consensyssoftware.atlassian.net/browse/LENS-44)

## Type of change

- [ ] Bug fix (non-breaking change, which fixes an issue)
- [ ] New feature (non-breaking change, which adds functionality)
- [x] Enhancement (non-breaking small changes to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

[LENS-44]: https://consensyssoftware.atlassian.net/browse/LENS-44?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by alainncls — APPROVED** [review]

**Comment by Akaryatrh** [comment]
Don't we have to duplicate the content modified from `packages/ui/src/*` to `apps/web/packages/ui/src/*` otherwise we will have issues on build phase?

**Review by Akaryatrh — CHANGES_REQUESTED** [review]
Don't we have to duplicate the content modified from `packages/ui/src/*` to `apps/web/packages/ui/src/*` otherwise we will have issues on build phase?

File: packages/ui/src/TabButton.tsx [hunk 1]
```diff
@@ -24,13 +24,13 @@ const TabButton: FC<TabButtonProps> = ({ name, icon, active, type, showOnSm = fa
         onClick();
       }}
       className={clsx(
-        { 'text-brand bg-brand-100 bg-opacity-100 dark:bg-opacity-20': active },
-        'hover:bg-brand-100 flex items-center space-x-2 rounded-lg px-4 py-2 text-sm font-medium text-gray-500 hover:bg-opacity-100 dark:hover:bg-opacity-20 sm:px-3 sm:py-1.5'
+        { 'text-brand bg-blue-500 bg-opacity-100 dark:bg-opacity-20': active },
+        'text-brand flex items-center space-x-2 rounded-[2px] px-4 py-2 text-sm font-medium hover:bg-blue-500 hover:bg-opacity-100 dark:hover:bg-opacity-20 sm:px-3 sm:py-1.5'
```
Comment (Akaryatrh) [inline packages/ui/src/TabButton.tsx:28]: I'd go with this instead
```suggestion
{ 'bg-brand-500 text-dark dark:text-dark hover:cursor-default dark:bg-white': active },
        'text-dark dark:hover:text-dark hover:bg-brand-500 flex items-center space-x-2 rounded-full px-4 py-2 text-sm font-medium dark:text-white sm:px-3 sm:py-1.5'
```
Capture d’écran 2023-05-25 à 14 02 19
Capture d’écran 2023-05-25 à 14 02 12

**Review by Akaryatrh — CHANGES_REQUESTED** [review]

File: apps/web/packages/ui/src/TabButton.tsx [hunk 2]
```diff
@@ -24,13 +24,19 @@ const TabButton: FC<TabButtonProps> = ({ name, icon, active, type, showOnSm = fa
...
-        { 'text-brand bg-brand-100 bg-opacity-100 dark:bg-opacity-20': active },
-        'hover:bg-brand-100 flex items-center space-x-2 rounded-lg px-4 py-2 text-sm font-medium text-gray-500 hover:bg-opacity-100 dark:hover:bg-opacity-20 sm:px-3 sm:py-1.5'
+        {
+          'text-dark dark:text-dark flex items-center space-x-2 rounded-full px-4 py-2 text-sm font-medium hover:cursor-default dark:bg-white sm:px-3 sm:py-1.5':
+            active
+        },
+        {
+          'text-dark dark:hover:text-dark hover:bg-brand-500 flex items-center space-x-2 rounded-full px-4 py-2 text-sm font-medium dark:text-white sm:px-3 sm:py-1.5':
+            !active
+        }
```
Comment (Akaryatrh) [inline apps/web/packages/ui/src/TabButton.tsx:34]: Still not ok to me 😅  here's the right styles IMO:
```suggestion
        {
          'text-dark bg-brand-500 flex items-center space-x-2 rounded-full px-4 py-2 text-sm font-medium hover:cursor-default sm:px-3 sm:py-1.5':
            active
        },
        {
          'text-dark dark:hover:text-dark hover:bg-brand-500 flex items-center space-x-2 rounded-full px-4 py-2 text-sm font-medium dark:text-white sm:px-3 sm:py-1.5':
            !active
        }
```

**Review by Akaryatrh — APPROVED** [review]

**Pull request merged**